src/backend/marsha/settings.py

```diff
@@ -863,6 +863,8 @@ class Development(Base):
     CACHES = {"default": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"}}
     STAT_BACKEND = values.Value("marsha.core.stats.dummy_backend")
 
+    # P2P
+    P2P_LIVE_WEB_TORRENT_TRACKER_URLS = values.ListValue(["http://localhost:8080"])
```

Comment:
this line should be deleted

Reply:
In fact I would like to provide in development the webtorrent tracker I made and added to docker-compose